docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -134,6 +134,12 @@
             <td>Integer</td>
             <td>Maximum number of retries when commit failed.</td>
         </tr>
+        <tr>
+            <td><h5>commit.max-timeout</h5></td>
```

Comment:
max timeout seems weird... it is just timeout?
`commit.timeout`

Reply:
Make sense，changed to commit.timeout，Thanks~ @JingsongLi